### ISSUE TITLE
Using pretrained inception network in inference mode for evaluating GANs while training the main model in DistributedDataParallel (DDP) mode

### ISSUE THREAD

**Issue by mshahbazi72** [pytorch/pytorch#51449]
I am using DistributedDataParallel to train a generative adversarial network. To evaluate the model during training, I need to compute the FID score using a pre-trained inception model as a feature extractor.
Apparently, a model that does not require gradient computation (the pre-trained inception net) cannot be passed to DistributedDataParallel. 
How can I run the inception model in inference mode on multiple GPUs during the training in DDP mode?

cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @agolynski @SciPioneer @H-Huang @mrzzd @cbalioglu

**Comment by mrshenli** [comment]
Hey @mshahbazi72 does wrapping the model forward pass using `with torch.no_grad():` work?

**Comment by mrshenli** [comment]
Also, could you please post questions to [PyTorch Forum](https://discuss.pytorch.org/c/distributed/12)? If this turns out to be a feature request or bug report, please feel free to open another issue here. Thanks!

**Comment by mshahbazi72** [comment]
Thanks @mrshenli. That solved the problem.
